FDB_NOTES.md

```diff
@@ -25,25 +24,25 @@ Things of Note
    this situation we don't run the prep_and_validate code on pre-fdb
    CouchDB. The new code always checks stubs before merging revision trees.
    I'm sure the old way would fail somehow, but it would fail further on
-   which means we may have failed with a different reason (conflict, etc)
+   which means we may have failed with a different reason (conflict, etc.)
    before we got to the next place we check for missing stubs.
 
 6. For multi-doc updates we'll need to investigate user versions on
-   versionstamps within a transaction. Also this likely prevents the
+   version stamps within a transaction. Also this likely prevents the
```

Comment:
Grammatically this is correct but in [FDB docs and API](https://apple.github.io/foundationdb/api-python.html?highlight=versionstamp#fdb.tuple.Versionstamp) they are usually referred to as a `versionstamp` all concatenated.

Reply:
HI, that makes sense, reverting...

Reply:
Looks great, thank you, for your contribution!

Reply:
Looks great, thank you, for your contribution!